bootable-jar/runtime/src/main/java/org/wildfly/core/jar/runtime/Arguments.java

```diff
@@ -101,6 +102,11 @@ private void handleArguments(List<String> args) throws Exception {
                 serverArguments.add(a);
             } else if (CommandLineConstants.HELP.equals(a)) {
                 isHelp = true;
+            } else if (a.startsWith("--cli-script")) {
+                cliScript = Paths.get(getValue(a));
```

Comment:
what would happen if the path of the script is relative? is it resolved based on the bootable jar path or some of our well known dirs?

Reply:
If relative, it is relative to the JVM work dir (from where the JVM has been launched).